menu_GC_Setup.sh

```diff
@@ -12,10 +12,7 @@ Press Enter to execute the highlighted option.\n" 17 50 7\
  "1" "Install Nightscout phase 1 - 15 minutes"\
  "2" "Install Nightscout phase 2 - 5 minutes"\
  "3" "Update platform"\
- "4" "Bootstrap the stable release"\
```

Comment:
What about the option to "Bootstrap the stable release". This still seems reasonable unless the vps-1 branch is going to be fully freezed.

Reply:
You are right, that will wok.  But, I have a different motive for this.

I am thinking the menus are too cluttered.  Not many people use the dev branch anyway.  If anyone wants to use either branch, they can by manually running the appropriate bootstrap.
So, one of my motives is to just remove bootstrap from the menu and have it only in the guide for the user to copy and paste into the terminal.

I was thinking that we will never put it back in the menu even for the Ubuntu 24 version.

The disadvantage will be that the user will need to exit the menu and paste a command in the terminal.
If someone wants to use the dev branch, I hope they should feel comfortable doing that.

Reply:
@tzachi-dar And thanks for the review

Reply:
This makes sense. Most people don't use the dev branch.